src/app/modules/main/chat_section/controller.nim

```diff
@@ -162,18 +166,33 @@ method setActiveItemSubItem*(self: Controller, itemId: string, subItemId: string
 
   self.delegate.activeItemSubItemSet(self.activeItemId, self.activeSubItemId)
 
-method removeActiveFromThisChat*(self: Controller, itemId: string) =
-  if self.activeItemId != itemId:
-    return
+method deleteChannel*(self: Controller, itemId: string) =
+  let allChats = self.communityService.getChats(self.getMySectionId())
+  self.activeSubItemId = ""
+  if allChats.len == 0:
+    self.activeItemId = ""
+  else:
+    self.activeItemId = self.getMySectionId() & allChats[0].id
+
+  self.communityService.deleteCommunityChat(self.getMySectionId(), itemId)
 
+method deleteChat*(self: Controller, itemId: string) =
   let allChats = self.chatService.getAllChats()
-
   self.activeSubItemId = ""
   if allChats.len == 0:
     self.activeItemId = ""
   else:
     self.activeItemId = allChats[0].id
 
+  echo "Todo for chat "
```

Comment:
This a left over?

Reply:
Hey Pascal. no not left over, the remove for 1x1 chat will also land here, its not implemented yet. but maybe I can turn that echo into a comment instead?